compiler/src/dotty/tools/dotc/transform/ReifyQuotes.scala

```diff
@@ -88,7 +94,7 @@ class ReifyQuotes extends MacroTransformWithImplicits {
   }
 
   override def run(implicit ctx: Context): Unit =
-    if (ctx.compilationUnit.containsQuotesOrSplices) super.run
+    if (ctx.compilationUnit.containsInlineCalls || ctx.compilationUnit.containsQuotesOrSplices) super.run
```

Comment:
Merge the two flags? Is one ever tested independently of the other?

Reply:
They should be merged, I forgot this.